package.json

```diff
@@ -201,7 +211,9 @@
     "copy-fonts": "python3 ./scripts/copy_fonts.py && npm run generate-icons",
     "hint": "node ./scripts/hints.js",
     "storybook": "storybook dev -p 6006 --no-open",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "jest": "jest --config ./jsapp/jest/jest.config.ts",
```

Comment:
We could add this to our GitHub CI (.github/workflows/npm-test.yml)
```yml
      # Run Jest component tests
      - name: Run Jest component tests
      - run: npx jest --config ./jsapp/jest/jest.config.ts --ci
```
Here, or in a separate PR.

Reply:
Good catch, I think that's in the scope of adding jest to project, because normally all tests are always run on CI.

Reply:
I was thinking about doing this in a different PR, after some more tests were added. But I believe there's no problem doing so here anyways. 👍🏻